scripts/find-breakpoints.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+OUTPUT=$(grep -R -i -n --include=\*.py "pdb.set_trace()" .)
```

Comment:
Looks like a generic thing for Python projects? Better in https://github.com/libero/scripts?

Reply:
Yes, this project is based on [python-project-poc](https://github.com/libero/python-project-poc) which provides most things needed to get started on python project, so everything python specific is living here. I had thought of creating a script or CLI cookie cutter for python projects but I wanted to work with this template and see what needs changing before doing so.